src/envs/otoole_convert.yaml

```diff
@@ -0,0 +1,7 @@
+name: otoole_convert
+channels:
+  - conda-forge
+dependencies:
+ - pip
+ - pip: 
+    - otoole==0.11.0
```

Comment:
As it is, it looks like the model will throw an error if someone has a more recent (>0.11.0) otoole version. If exact otoole version 0.11.0 is recommended to generate the data for the Canada-US model, it should probably be specified in a README somewhere as well.

Reply:
Is there a newer version then 0.11.0? I think I (successfully) tested it on 0.10.0 as well, cause there is sometime backwards compatibility issues. Nevertheless, good idea! Did you want to add it in?

Reply:
It does seem to work with otoole 0.10.0, and I don't know why. It's odd that we'd specify a version if it works with older versions anyways, but it's functional, so I think we can leave it be, if you think it's okay.

Reply:
Its just for when you create the conda environment using the `--use-conda` flag